clients/admin-ui/src/features/config-wizard/constants.tsx

```diff
@@ -46,10 +46,16 @@ export const iconButtonSize = 107;
 
 // When more links like these are introduced we should move them to a single file.
```

Comment:
This mentions moving these links to a different file, but I think they fit nicely here?